english/eating-walls/find-a-toilet/door/open-door.md

```diff
@@ -4,4 +4,6 @@ You jump back as you see dark black eyes.
 
 Do you:
 
-[Stand still in terror](bear/stand-still.md)
\ No newline at end of file
+[Stand still in terror](bear/stand-still.md)
+
+[Run into the room and close the door](bear/run-into-room.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.